backend/ipip/ipip_network.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2017 flannel authors
```

Comment:
Much of this file is the same as the host-gw code. It would be great if the common code could be pulled out (and then it could be shared by the vxlan code too but I would do that in a followup PR).

Reply:
I'll try to do it

Reply:
Added RouteNetwork in common.go.